libpip2pi/commands.py

```diff
@@ -227,7 +227,7 @@ def dir2pi(argv=sys.argv, use_symlink=OS_HAS_SYMLINK):
         if pkg_basename.startswith("."):
             continue
         pkg_name, pkg_rest = file_to_package(pkg_basename, pkgdir)
-        pkg_dir = pkgdirpath("simple", pkg_name)
+        pkg_dir = pkgdirpath("simple", pkg_name.lower())
```

Comment:
Maybe there should be a replace('_', '-') to handle underscores to hyphens in addition to lower() for normalizing the directory names. Packages that have underscores in them still receive the deprecation warning.